README.md

```diff
@@ -8,6 +8,7 @@ COM Examples contains sample implementations that use Microsoft COM technology.
 * [AtlServer](AtlServer#atlhenlib): An ATL implementation of a COM server that provides hens
 * [AtlFreeServer](AtlFreeServer#atlfreeserver): An COM server implemented without ATL. It provides dogs
 * [ManagedServer](ManagedServer/): An COM server implemented in .NET. Here you can buy dogs.
+* [PyComServer](PyComServer/): An Com server implemented in python. It provides snakes.
```

Comment:
Please also update the Build section with necessary dependencies
* Does Python support always come with Visual Studio, or does the programmer have to opt in to install some python tools for Visual Studio?
* Python is needed. Maybe specify a minimum version